### PR TITLE
Rebase: Interjections should not force preanimation if 'Pre' is unchecked

### PR DESCRIPTION
**Pull request by in1tiate**
Rebase of #133 to current master.

### CODE REVIEW

**Review by oldmud0 — APPROVED** [review]
TODO: update documentation

File: src/courtroom.cpp [hunk 1]
```diff
@@ -1597,13 +1596,11 @@ void Courtroom::on_chat_return_pressed()
...
-      if (f_emote_mod == 4 || f_emote_mod == 5)
-        f_emote_mod = 6;
-      else
-        f_emote_mod = 2;
-    }
+  if (objection_state > 0 && ui_pre->isChecked()) {
+    if (f_emote_mod == 4 || f_emote_mod == 5)
+      f_emote_mod = 6;
+    else
+      f_emote_mod = 2;
```
Comment (Crystalwarrior) [inline src/courtroom.cpp:1603]: wait does the logic actually change in any way, isn't it just condensing an if check but the inherent logic is the same? Has this been tested to work?

**Review by Crystalwarrior — COMMENTED** [review]
@in1tiate test? work good?

File: src/courtroom.cpp (hunk 1 above)
Comment (in1tiate) [reply src/courtroom.cpp:1603]: I honestly don't remember what I changed here to make this work, but I just tested it and it _does_ work, so make of that what you will

**Review by Crystalwarrior — APPROVED** [review]
It is a mystery why this works

**Comment by oldmud0** [comment]
Until there is an explanation for why this works, I'm going to treat this is as if it were a formatting commit.

**Pull request merged**